### DISCUSSION TITLE
REI works then stops 1.19.0

### DISCUSSION THREAD

**Discussion by richie19942** [shedaniel/RoughlyEnoughItems#1107] [Q & A]
When I first load the modpack Medieval MC REI works as intended then randomly stops, @(modpack) still throws results but a simple search for an item eg. cobble doesn't show. Deleting the mod pack and reinstalling works again temporarily. I tried updated to latest REI in the gamefiles and deleted the old with no avail. Ive also tried deleting the config folder and re loading the game, also didn't work.

It show REI caching as I type then does nothing after that message dissapears.

Worth while notes: Others on the server don't appear to have the same problem with the exact same modpack.

Any suggestions?

**Comment by shedaniel** [comment]
Hey, thanks for creating a discussion to RoughlyEnoughItems. To help trace the issue down, please include the necessary logs

You should upload the `latest.log` from the `.minecraft/logs` folder that is next to your `mods` folder.

**Reply by richie19942** [reply]
[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9463839/latest.log)
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/9463841/rei.log)
